### ISSUE TITLE
YouTube API connection is broken

### ISSUE THREAD

**Issue by shiffman** [shiffman/shiffman.net#74]
<img width="1050" alt="screen shot 2019-02-10 at 9 45 28 pm" src="https://user-images.githubusercontent.com/191758/52544201-3a130e00-2d7d-11e9-969b-b17d5d093b6f.png">

**Comment by sirluky** [comment]
![image](https://user-images.githubusercontent.com/38957065/52820305-ec174680-30ab-11e9-8cd7-c79c8af7f5be.png)
this is the problem

**Comment by shiffman** [comment]
@phucduong86 Is the error that @sirluky mentions above still relevant? . . or is simply a matter of me going over rate limits as you described in a recent e-mail? Perhaps I'm using an old way to communicate with YouTube here as I don't have these issues at https://thecodingtrain.com

**Comment by phucduong86** [comment]
@shiffman 
1. The above error is no longer relevant (it refers to an API key that has been removed). We still get an error from the API but it is just rate limits issue now.
2. I took a quick look at thecodingtrain's code and it is actually not using the API at all. Each time you have a new video you or a contributor add a post with the video's metadata (eg 142.1-rubiks-cube.md_ and the page uses the video_id from this post to render the video card.
![image](https://user-images.githubusercontent.com/21692768/55331901-44d74e80-5462-11e9-8b3c-1a9a60d59584.png)
